lib/components/resource-editing/resource-form/FileInput.js

```diff
@@ -161,7 +161,7 @@ var FileInput = React.createClass({
 				changeHandler = this._handleChange;
 
 			// Read each file of array, b64 encode, and bubble up to file select handler
-			files = files.map(function (file, index) {
+			files = files.forEach(function (file, index) {
```

Comment:
This isn't working right. For each doesn't return a value to be assigned to `files`